src/main/java/com/aws/greengrass/shadowmanager/ShadowManager.java

```diff
@@ -321,7 +321,6 @@ protected void install(InstallConfig installConfig) {
                         try {
```

Comment:
^^
For all these configs, can we just do a read-through instead of copying data into fields when there is a change?

Much simpler and more correct if we can just read the currently configured value (or default) when it is needed. Then there's no possibility of a discrepancy between what we're using for the value and what was configured by the user.

Different PR though.

Reply:
yeah - wanted that as a separate PR